README.md

```diff
@@ -16,25 +17,25 @@ integrating and optimizing their compute engines.
 Velox provides the following high-level components:
 
 * **Type**: a generic typing system that supports scalar, complex, and nested
-  types, such as structs, maps, arrays, tensors, etc.
+  types, such as structs, maps, arrays, etc.
 * **Vector**: an [Arrow-compatible columnar memory layout
   module](https://facebookincubator.github.io/velox/develop/vectors.html),
-  which provides multiple encodings, such as Flat, Dictionary, Constant,
-  Sequence/RLE, and Bias, in addition to a lazy materialization pattern and
-  support for out-of-order writes.
+  providing encodings such as Flat, Dictionary, Constant, and Sequence/RLE, in
```

Comment:
do we support `Sequence/RLE` encodings ?

Reply:
We do have it, but it's not really plumbed into anything today. We have had many on and off discussions on whether we should it, so it's not unrealistic to think we will pick it up in the future. 

https://github.com/facebookincubator/velox/blob/main/velox/vector/SequenceVector.h